packages/perseus/src/widgets/expression.tsx

```diff
@@ -642,20 +644,20 @@ const keypadConfigurationForProps = (
 
     // TODO(charlie): Alert the keypad as to which of these symbols should be
     // treated as functions.
-    const extraVariables = Object.keys(uniqueExtraVariables);
+    const extraVariables: Key[] = Object.keys(uniqueExtraVariables) as Key[];
```

Comment:
Necessary changes to support the new return type.

Reply:
This seems suspicious to me; if we need to coerce this at all, I think it needs to be `ReadonlyArray<Key>` rather than `Key[]`. I imagine the source of the issue is elsewhere though...

Oh this might be part of the problem:

``` JS
    if (!extraKeys.length) {
        // If there are no extra symbols available, we include Pi anyway, so
        // that the "extra symbols" button doesn't appear empty.
        extraKeys.push("PI");
    }
```

We won't be using the long-press "extra symbols" button in v2 keypad (though we still need it while v1 is alive). 🤔 Anyway, I think we could still figure out a way to make it `ReadonlyArray<Key>`, otherwise I wonder if this is a net loss: we made `keypadConfigurationForProps` more explicit, but had to add a bunch of `Key[]` which we typically try to avoid.